app/src/main/AndroidManifest.xml

```diff
@@ -2,22 +2,26 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="com.example.fithub">
 
+    <uses-permission android:name="android.permission.INTERNET"/>
+    
     <application
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.FitHub">
+        <activity android:name=".Register"/>
         <activity
-            android:name=".MainActivity"
```

Comment:
The loading screen will be the launcher activity which will lead to the Login. I will change this after these files are merged.

Reply:
@notlydia @ioavliv this commit has conflicts. can't merge them before resolving